tools/fuchsia/build_fuchsia_artifacts.py

```diff
@@ -149,10 +152,20 @@ def CopyToBucket(src, dst, product=False):
   CopyToBucketWithMode(src, dst, True, product, 'dart')
 
 
+def CopyDepsToBucket(src, dst, arch):
```

Comment:
could you add Vulkan to the name of this method? Maybe `CopyVulkanDepsToBucket`?